enterprise/server/remote_execution/containers/ociruntime/BUILD

```diff
@@ -47,5 +51,17 @@ go_test(
         "//server/util/testing/flags",
         "@com_github_stretchr_testify//assert",
         "@com_github_stretchr_testify//require",
+        "@io_bazel_rules_go//go/runfiles:go_default_library",
     ],
 )
+
+genrule(
+    name = "crun_copy",
```

Comment:
I think that this could just be an `alias`

Reply:
nice - done